requirements.txt

```diff
@@ -0,0 +1,13 @@
+# git+git://github.com/oasis-open/cti-pattern-matcher.git@b265862971eb63c04a8a054a2adb13860edf7846#egg=stix2-matcher #uncomment when running locally
```

Comment:
This file shouldn't be included here, the requirements are generated by a script.